lib/cylc/gui/SuiteControlTree.py

```diff
@@ -238,6 +238,18 @@ def _set_tooltip( self, widget, tip_text ):
         tip.enable()
         tip.set_tip( widget, tip_text )
 
+    def _set_cell_text_time(self, column, cell, model, iter_, n):
+        """Remove the date part if it matches the last update date."""
+        if "T" in self.updater.dt:
+            last_update_date = self.updater.dt.split("T")[0]
+        else:
+            last_update_date = None
+        if last_update_date is not None:
+            date_time_string = model.get_value(iter_, n)
+            date_time_string = date_time_string.replace(last_update_date
+                                                        + "T", "", 1)
+        return cell.set_property("text", date_time_string)
```

Comment:
Need to keep the `markup` setting for column 8.

(the italicised Tfinish with the question mark.)